docs/node/run-your-node/paratime-node.mdx

```diff
@@ -306,11 +272,12 @@ registration:
     entity_id: {{ entity_id }}
 
 runtime:
-    paths:
-        # Paths to ParaTime bundles for all of the supported ParaTimes.
-        - {{ runtime_orc_path }}
-    # The following section is required for ParaTimes which are running inside
-    # the Intel SGX Trusted Execution Environment.
+    # Runtime configuration for every ParaTime. Mandatory for ParaTimes with
+    # hot-loading support (e.g. Sapphire, Cipher).
+    runtimes:
+        - id: {{ runtime_identifier }}
```

Comment:
```suggestion
        - id: {{ runtime_identifier_1 }}
        - id: {{ runtime_identifier_2 }}
```

```suggestion
        - id: {{ runtime_identifier }}
        # List additional ParaTimes here.
```

Is it obvious that we can list multiple identifiers here, or is this change better/worse (or some other)?